.github/ENV_SETUP_INSTRUCTION.md

```diff
@@ -171,6 +171,9 @@ Follow the rest of the setup instructions (providing the environment credentials
 
 **---**
 
+**Note : If you got stuck while setting up the development environment, then don't worry 😉 You may check out this setup tutorial 👇 :**
+https://youtu.be/yha9CNMZV5k
```

Comment:
@Aryamanz29 . This looks good to me already. What do you think if we take it a little step further by embedding the youtube image in here instead of just url link? [Here](https://www.nathan-lam.com/blog/embed-video-in-github-readme.html)'s an article that gives various options on how to do this. I'm happy to whichever one you choose from the options 😉 .

Reply:
@mtreacy002  Done 👍 , I've used a youtube thumbnail image.